core/src/main/resources/reference.conf

```diff
@@ -152,12 +152,30 @@ google {
 }
 
 docker {
-  // Set this to match your available quota against the Google Container Engine API
-  gcr-api-queries-per-100-seconds = 1000
-  // Time in minutes before an entry expires from the docker hashes cache and needs to be fetched again
-  cache-entry-ttl = "20 minutes"
-  // Maximum number of elements to be kept in the cache. If the limit is reached, old elements will be removed from the cache
-  cache-size = 200
+  hash-lookup {
+    #######################################################################################
+    ##################################### ATTENTION #######################################
+    #######################################################################################
+    # If call caching is enabled, most user will want to keep this TRUE,                  #
+    # as setting it false will open the door for potential false positives.               #
+    # When true, hashes for docker images will be looked up and the hash will be used for #   
+    # caching purposes as well as for running the job itself.                             #
+    # When false, hashes for docker images will NOT be looked up and the value defined    #
+    # in the WDL file will be used as is.                                                 #
+    enabled = true                                                                        #
+    #######################################################################################
+    
+    // Set this to match your available quota against the Google Container Engine API
+    gcr-api-queries-per-100-seconds = 1000
+    // Time in minutes before an entry expires from the docker hashes cache and needs to be fetched again
+    cache-entry-ttl = "20 minutes"
+    // Maximum number of elements to be kept in the cache. If the limit is reached, old elements will be removed from the cache
+    cache-size = 200
+    // How should docker hashes be looked up. Possible values are "local" and "remote"
+    // "local": Lookup hashes on the local docker daemon using the cli
+    // "remote": Lookup hashes on docker hub and gcr
+    method = "local"
```

Comment:
My thinking was if the backend is changed away from the default, the user would need to know to change this value to keep the server from wasting time doing local Docker lookups.